src/ray/common/bundle_spec.cc

```diff
@@ -31,10 +31,23 @@ const ResourceSet &BundleSpecification::GetRequiredResources() const {
   return *unit_resource_;
 }
 
-BundleID BundleSpecification::BundleId() const {
-  if (message_->bundle_id().empty() /* e.g., empty proto default */) {
-    return BundleID::Nil();
+UniqueID BundleSpecification::BundleId() const {
```

Comment:
Can we use `std::pair<PlacementGroupID, int64_t>`?

Reply:
I find I need a string when i schedule the resource or print the log.
BTW, the pair can't be set as key in unordered_map directly. (Maybe I change it to flat_map can fix it?)
Can I give two api named BundleIDAsString and BundleIDAsPair?